.travis.yml

```diff
@@ -38,8 +38,7 @@ matrix:
         - bash utils/ci/script.sh
         # remove cached documentation, otherwise files from previous PRs can get included
         - rm -rf target/doc
-        - cargo doc --no-deps --all --all-features
-        - cargo deadlinks --dir target/doc
+        - cargo deadlinks --ignore-fragments -- --all-features --all
```

Comment:
Let's make sure to backport this to GitHub actions.

Reply:
It's done.